Pod/Classes/ViewControllers/ARPartnerComponentViewController.m

```diff
@@ -0,0 +1,32 @@
+#import "ARPartnerComponentViewController.h"
+#import "ARGraphQLQueryPreloader.h"
+
+@implementation ARPartnerComponentViewController
+
++ (NSDictionary *)propsWithPartnerID:(NSString *)partnerID;
+{
+    BOOL isPad = UI_USER_INTERFACE_IDIOM() == UIUserInterfaceIdiomPad;
+    return @{ @"partnerID": partnerID, @"isPad": @(isPad) };
```

Comment:
We can use the existing `isPad` function in [`hardware.ts`](https://github.com/artsy/emission/blob/master/src/lib/Scenes/Artwork/hardware.ts) (though we may want to move that file to a higher-level directory).